crates/iota-transaction-builder/src/lib.rs

```diff
@@ -63,55 +67,16 @@ pub trait DataReader {
 
 #[derive(Clone)]
 pub struct TransactionBuilder(Arc<dyn DataReader + Sync + Send>);
+ 
+use stake::TransactionBuilder::{request_add_stake, request_withdraw_stake, request_add_timelocked_stake, request_withdraw_timelocked_stake};
+use utils::TransactionBuilder::{select_gas, get_object_arg, input_refs, resolve_and_checks_json_args, get_object_ref, get_object_ref_and_type};
+use package::TransactionBuilder::{publish_tx_kind, publish, upgrade_tx_kind, upgrade};
```

Comment:
These should be moved further up together before the `pub trait DataReader`, together with all the other `use` statements

Reply:
sounds good, i'll do that now.